File: tests/test_data/ChatGPT/v1/Transcribed/Adivina.py
import random

def adivina():
    numero_secreto = random.randint(1, 10)
    try:
        intento = int(input("Adivina el número (entre 1 y 10): "))
        if intento == numero_secreto:
            print("¡Correcto! Adivinaste el número.")
        else:
            print(f"Incorrecto, el número era: {numero_secreto}")
    except ValueError:
        print("Entrada inválida, debe ser un número entero.")

adivina()

File: tests/test_data/ChatGPT/v1/Transcribed_JS/Adivina.js
const random = Math.floor(Math.random() * 10) + 1;
let intento = prompt("Adivina el número (entre 1 y 10): ");

if (intento == random) {
    console.log(`¡Correcto! Adivinaste el número.`);
} else {
    console.log(`Incorrecto, el número era: ${random}`);
}